### PR TITLE
Copied the perftest settings for nfdiv to setup the sandbox environment.

### PR DESCRIPTION
**Pull request by Nooshu**
### Jira link (if applicable)
[DTSSE-5068](https://tools.hmcts.net/jira/browse/DTSSE-5068)

### Change description ###
Copied the perftest settings for nfdiv to set up the sandbox environment for cache and compression testing.

### Checklist
- [x] commit messages are meaningful and follow good commit message guidelines

### Added
-  `apps/nfdiv/sbox/00/kustomization.yaml`
  - Nfdiv namespace created with base as a resource.
-  `apps/nfdiv/sbox/01/kustomization.yaml`
  - Nfdiv namespace created with base as a resource.
-  `apps/nfdiv/sbox/base/kustomization.yaml`
  - Kustomization file added for nfdiv sbox base, includes resources and patches.
-  `apps/nfdiv/serviceaccount/sbox.yaml`
  - Service account file for sbox environment added with metadata and annotations.

### CODE REVIEW

**Review by thomast1906 — CHANGES_REQUESTED** [review]
A lot of perftest.yaml references, can you update/create for sbox?

**Comment by Nooshu** [comment]
> A lot of perftest.yaml references, can you update/create for sbox?

Thanks, @thomast1906! Done 👍

**Review by thomast1906 — CHANGES_REQUESTED** [review]
you have to create folder/structure for sbox.yaml etc, reach out on slack to discuss

File: apps/nfdiv/sbox/base/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - path: ../../nfdiv-cron-remind-applicant2/sbox.yaml
+  - path: ../../nfdiv-cron-progress-to-awaiting-final-order/sbox.yaml
+  - path: ../../nfdiv-cron-process-failed-to-unlink-cases/sbox.yaml
+  - path: ../../nfdiv-cron-final-order-overdue/sbox.yaml
+  - path: ../../nfdiv-cron-find-cases-with-successful-payments/sbox.yaml
+  - path: ../../nfdiv-cron-eligible-for-switch-to-sole/sbox.yaml
+  - path: ../../nfdiv-cron-js-disputed-answer-overdue/sbox.yaml
+  - path: ../../nfdiv-cron-partner-not-applied-for-final-order/sbox.yaml
+  - path: ../../nfdiv-cron-bulk-list-create/sbox/sbox.yaml
+  - path: ../../nfdiv-cron-migrate-bulk-cases/sbox/sbox.yaml
```
Comment (thomast1906) [inline apps/nfdiv/sbox/base/kustomization.yaml:56]: remove all cron patches

File: apps/nfdiv/sbox/base/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../nfdiv-cron-progress-held-cases/nfdiv-cron-progress-held-cases.yaml
+  - ../../nfdiv-cron-remind-applicants-apply-for-fo/nfdiv-cron-remind-applicants-apply-for-fo.yaml
+  - ../../nfdiv-cron-notify-applicants-apply-co/nfdiv-cron-notify-applicants-apply-co.yaml
+  - ../../nfdiv-cron-process-cases-to-be-removed/nfdiv-cron-process-cases-to-be-removed.yaml
+  - ../../nfdiv-cron-process-failed-scheduled-cases/nfdiv-cron-process-failed-scheduled-cases.yaml
```
[the remark this answers is not in the document]
Comment (Nooshu) [reply apps/nfdiv/sbox/base/kustomization.yaml]: Thanks, completed

File: apps/nfdiv/sbox/base/kustomization.yaml (hunk 1 above)
Comment (Nooshu) [reply apps/nfdiv/sbox/base/kustomization.yaml]: Thanks, completed

**Comment by Nooshu** [comment]
Closing PR, divorce is already on perftest.

**Pull request closed without merging**